apps/dashboard/src/contract-ui/tabs/tokens/components/transfer-button.tsx

```diff
@@ -141,7 +135,8 @@ export const TokenTransferButton: React.FC<TokenTransferButtonProps> = ({
                     label: "error",
                     error,
                   });
-                  onError(error);
+                  toast.error("Failed to transfer tokens");
+                  console.error(error);
                 },
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +120 to +140]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->use `toast.promise` instead on `sendConfirmation.mutateAsync()`, this way you get a loading state and a success and error state that is consistent<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4661?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->